terrautils/sensors.py

```diff
@@ -118,75 +121,94 @@
+        },
+
+        'hyperspectral2nc_swir': {
+            'display': 'SWIR Hyperspectral NetCDFs',
+            'template': '{base}/{station}/Level_1/'
+                        '{sensor}/{date}/{time}/{filename}',
+            'pattern': '{sensor}_lv1_{station}_{timestamp}{opts}.nc',
         },
 
         'stereoTop_geotiff': {
+            'display': 'stereoTop GeoTIFFs',
             'template': '{base}/{station}/Level_1/'
                         '{sensor}/{date}/{time}/{filename}',
             'pattern': '{sensor}_lv1_{station}_{timestamp}{opts}.tif',
         },
 
         'stereoTop_canopyCover': {
+            'display': 'stereoTop Canopy Cover',
             'template': '{base}/{station}/Level_1/'
                         '{sensor}/{date}/{time}/{filename}',
             'pattern': '{sensor}_lv1_{station}_{timestamp}{opts}.tif',
         },
 
         'texture_analysis': {
+            'display': 'stereoTop R Texture Analysis',
             'template': '{base}/{station}/Level_1/'
                         '{sensor}/{date}/{time}/{filename}',
             'pattern': '{sensor}_lv1_{station}_{timestamp}{opts}.csv',
         },
 
         'flir2tif': {
+            'display': 'flirIrCamera GeoTIFFs',
             'template': '{base}/{station}/Level_1/'
                         '{sensor}/{date}/{time}/{filename}',
             'pattern': '{sensor}_lv1_{station}_{timestamp}{opts}.tif',
         },
 
         'EnvironmentLogger': {
+            'display': 'EnvironmentLogger netCDFs',
             'template': '{base}/{station}/Level_1/'
                         '{sensor}/{date}/{filename}',
             'pattern': '{sensor}_lv1_{station}_{timestamp}{opts}.nc',
         },
 
         'soil_removal_vnir': {
+            'display': 'VNIR Soil Removal Masks',
             'template': '{base}/{station}/Level_1/'
                         '{sensor}/{date}/{time}/{filename}',
             'pattern': 'VNIR_lv1_{station}_{timestamp}{opts}.nc'
         },
 
         'soil_removal_swir': {
+            'display': 'SWIR Soil Removal Masks',
             'template': '{base}/{station}/Level_1/'
                         '{sensor}/{date}/{time}/{filename}',
             'pattern': 'SWIR_lv1_{station}_{timestamp}{opts}.nc'
         },
 
         'scanner3DTop_mergedlas': {
+            'display': 'scanner3D Merged LAS',
             'template': '{base}/{station}/Level_1/'
                         'scanner3DTop/{date}/{time}/{filename}',
             'pattern': 'scanner3DTop_lv1_{station}_{timestamp}'
                        '_merged{opts}.las'
         },
 
         'scanner3DTop_plant_height': {
+            'display': 'scanner3D Plant Height',
             'template': '{base}/{station}/Level_1/'
                         'scanner3DTop/{date}/{time}/{filename}',
             'pattern': 'scanner3DTop_lv1_{station}_{timestamp}'
                        '_height{opts}.npy'
         },
 
         'scanner3DTop_heightmap': {
```

Comment:
another outdated one, I already updated this extractor to write to .tif instead of .bmp (bitmap, think old-school MS paint image format) and the name is updated.